AccessibilityInsightsForAndroidService/.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="JDK" project-jdk-type="JavaSDK">
```

Comment:
Is this intended as part of this PR? https://stackoverflow.com/q/58081993 suggests that it's probably an artifact of whether you/whoever generated this initially is developing on a mac or not, and one of the comments there suggests that it'll maybe stop flip-flopping if everyone is on Android Studio 3.6+; are you on an up to date android studio?

Reply:
Good catch, my android studio was out of date which was causing some .idea files to change. I've changed them back to the old versions.